validator/src/main/java/com/amazon/aoc/validators/AlarmPullingValidator.java

```diff
@@ -21,6 +23,11 @@
   private Integer pullDuration;
   private Integer pullTimes;
   private CloudWatchAlarmService cloudWatchAlarmService;
+  private CloudWatchService cloudWatchService;
+
+  private static final String SOAKING_NAMESPACE = "AWSOtelCollector/SoakingTest";
```

Comment:
can this namespace be sent from terraform to validator instead of define a constant here?